core/static/core/scss/emlo-common.scss

```diff
@@ -183,15 +184,46 @@ tr.selectable_entry:hover {
 
 .multi_rel_recref_container {
   display: grid;
-  grid-template-columns: 65% 35%;
+  grid-template-columns: 1fr 2fr;
   grid-row-gap: 8px;
-  width: 70%;
+  width: 100%;
+  font-size: $typescale-7;
 
   & > .grid-item {
     background-color: #f2f2f2;
   }
 }
 
+.sub-recref-form {
+  display: grid;
+  grid-template-columns: 1fr 2fr;
+  width: 100%;
+
+  input[type=date] {
+    width: 100px;
+  }
+
+  div:only-child {
+    width: 500px;
+  }
+}
+
+
 .autodate-div .from-div {
   margin-right: 20px;
+}
+
+#toc-body {
+  max-width: 400px;
+  background-color: #DCDFE5;
+  font-size: $typescale-5a;
+
```

Comment:
for `#toc-body` I suggest we add

`padding: 1em;`
to make it look a little nicer


 `min-width: 260px;`
 
 to prevent box from changing size when longest text becomes bold

![image](https://user-images.githubusercontent.com/5488106/201943196-3f9cf577-e9a2-4291-9132-e1f6ceb47bf9.png)